core/modules/system/system.theme.inc

```diff
@@ -171,19 +171,18 @@ function theme_system_modules_fieldset($variables) {
     // Add the third column: module version number.
     $row[] = array('class' => array('version'), 'data' => backdrop_render($module['version']));
 
-    if ($requires || $required_by) {
-      $requirements .= ' <div class="requirements">';
-      if ($requires) {
-        $requirements .= '<div class="admin-requirements">' . t('Requires: !module-list', array('!module-list' => implode(', ', $module['#requires']))) . '</div>';
-      }
-      if ($required_by) {
-        $requirements .= '<div class="admin-requirements">' . t('Required by: !module-list', array('!module-list' => implode(', ', $module['#required_by']))) . '</div>';
-      }
-      $requirements .= '</div>';
-
-      // Build a requirements toggle.
-      $toggle .= ' <a class="requirements-toggle" href="#">' . t('more') . '</a>';
+    $requirements .= ' <div class="requirements">';
+    if ($requires) {
+      $requirements .= '<div class="admin-requirements">' . t('Requires: !module-list', array('!module-list' => implode(', ', $module['#requires']))) . '</div>';
+    }
+    if ($required_by) {
+      $requirements .= '<div class="admin-requirements">' . t('Required by: !module-list', array('!module-list' => implode(', ', $module['#required_by']))) . '</div>';
     }
+    $requirements .= '<div class="admin-requirements">' . t('Tags: <span class="module-tags">!tags</span>', array('!tags' => backdrop_render($module['tags']))) . '</div>';
```

Comment:
Shouldn't we check that `$module['tags']` is not empty first ?

Reply:
Yes, good catch. If a project has neither a project or a tag this could be empty.

Reply:
Checking further, if there are no tags then "other" is automatically assigned, just like with packages. There will always be tags.